dffml/base.py

```diff
@@ -248,7 +253,16 @@ def config_get(cls, config, above, *path) -> BaseConfig:
             if "nargs" in arg:
                 value = list(map(arg["type"], value))
             else:
-                value = arg["type"](value)
+                # instantiation of generic collections are not allowed
+                try:
+                    value = arg["type"](value)
+                except TypeError:
+                    _to_cast = arg["type"].__dict__.get("_name").lower()
+                    if _to_cast == "type":
+                        value = locate(value)
+                    else:
+                        value = eval(f"{_to_cast}(value)")
```

Comment:
We need to avoid using `eval` as that can be a potential attack vector from a security standpoint

Reply:
I think this is for: https://github.com/intel/dffml/pull/253/files#diff-abe55247d9355e64411283cfc723040eR146

I'm pretty sure this was the reason I made this: https://github.com/intel/dffml/blob/a09d12b8c63659df734f7fd2864d799618c9a3a1/dffml/util/data.py#L72-L101

Used here: https://github.com/intel/dffml/blob/a09d12b8c63659df734f7fd2864d799618c9a3a1/dffml/df/types.py#L64-L92

You might want to replace the locate with the call to `type_lookup` and then in `BaseConfigurable.args()` use the `export_value` method. Also you'd need to add `Type` to `STANDARD_TYPES` mapped to `"type"`.

What those methods do is allow us to map Python types to programming language agnostic types. This way we can set up a similar `dump_to_string <-> load_from_string`  in implementations in other languages, where the typenames may be different.